frontend/demo/src/pages/Channels/index.tsx

```diff
@@ -0,0 +1,117 @@
+/* global FB */
+import React, {useCallback, useEffect, useState} from 'react';
+import {connect, ConnectedProps} from 'react-redux';
+import {RouteComponentProps} from 'react-router-dom';
+import FacebookLogin from 'react-facebook-login';
+import {Button} from '@airyhq/components';
+
+import {AiryConfig} from '../../api/airyConfig';
+import {Channel} from '../../model/Channel';
+import {getChannels, exploreChannels, connectChannel, disconnectChannel} from '../../actions/channel';
+import {StateModel} from '../../reducers/index';
+
+import styles from './index.module.scss';
+
+const mapDispatchToProps = {
+  getChannels,
+  exploreChannels,
+  connectChannel,
+  disconnectChannel,
+};
+
+const mapStateToProps = (state: StateModel) => {
+  return {
+    channels: state.data.channels,
+  };
+};
+
+const connector = connect(mapStateToProps, mapDispatchToProps);
+
+type ChannelsConnectProps = {} & ConnectedProps<typeof connector> & RouteComponentProps;
+
+const Channels = (props: ChannelsConnectProps) => {
+  const [facebookToken, setFacebookToken] = useState('');
+  useEffect(() => {
+    props.getChannels();
+  }, []);
+
+  const connect = token => {
```

Comment:
let's try to strong type everything. Pretty sure we will miss some, but let's try